pygmt/tests/test_clib_put.py

```diff
@@ -0,0 +1,197 @@
+"""
```

Comment:
What about separating the tests into three files:

- test_clib_put_vector.py
- test_clib_put_matrix.py
- test_clib_put_strings.py

Reply:
Yes, that can be arranged.